gatsby_fp/src/pages/404.js

```diff
@@ -5,7 +5,7 @@ import SEO from "../components/seo"
 
 const NotFoundPage = () => (
   <Layout>
-    <SEO title="404: Not found" />
+    <SEO title="Not Found - Stevens Institute of Technology" />
```

Comment:
None blocking - this should be under its own commit. So basically, you would have the following
- One that changes the title in layout.js
- One that changes the title in 404.js